guides/v2.3/graphql/search-pagination.md

```diff
@@ -20,7 +20,7 @@ The `search` element is optional, but it can be used with or without filters. Ea
 
 The `filter` element defines which search criteria to use to find the desired results. As with a REST call, each filter defines the field to be searched, the condition type, and the search value.
 
-Search filters are logically ANDed unless an `or` statement is specified. The search query can contain unlimited number of nested `or` clauses. However, you cannot perform a logical `or` across two `and` clauses, such as (A AND B) OR (X AND Y).
+Search filters are logically ANDed unless an `OR` statement is specified. The search query can contain unlimited number of nested `OR` clauses. However, you cannot perform a logical `OR` across two `AND` clauses, such as (A AND B) OR (X AND Y).
```

Comment:
The `or`s are actually used in GraphQL queries,  while the `and` is not. I prefer that you back out the `or` to `OR` changes. Remove the backticks around AND, since it's not code.

Reply:
its done :)